app/views/admin/basic_projects/_form.html.slim

```diff
@@ -0,0 +1,46 @@
+= simple_form_for([:admin, @project],
+    html: { \
+      class: 'form-horizontal organization-record',
+    },
+    wrapper: :horizontal_form_condensed,
+    defaults: { \
+      input_html: {class: 'form-element form-control'},
+    },
+  ) do |f|
+
+  = error_notification(f)
+
+  = f.input :name
+    .view-element = @project.try(:name)
+    = f.input_field :name
+
+  = f.input :status
+    .view-element = @project.try(:status)
+    = f.input_field :status_value, collection: BasicProject.status_options, include_blank: true
+
+  = authorized_form_field(simple_form: f, model: @project, field_name: :primary_agent,
+    choices: @agent_choices, classes: 'hidden-print')
+  = f.input t('activerecord.attributes.basic_project.primary_agent'), wrapper_html: {class: 'visible-print-block'}
+    span = @project.primary_agent.try(:name)
+
+  = authorized_form_field(simple_form: f, model: @project, field_name: :secondary_agent,
+    choices: @agent_choices, classes: 'hidden-print')
+  = f.input t('activerecord.attributes.basic_project.secondary_agent'), wrapper_html: {class: 'visible-print-block'}
+      span = @project.secondary_agent.try(:name)
+
+  = f.input :signing_date, label: t("activerecord.attributes.basic_project.start_date")
```

Comment:
Same here. I think the explicit label can be removed. Just change the key in the en.yml file to match the field name.